### PR TITLE
Fix new dm

### PR DESCRIPTION
**Pull request by seavan**
cleaning up unused code
fixes for bugs which Eren found
sidebar toggle state moved to ui-state

### CODE REVIEW

File: src/stores/ui-store.js [hunk 1]
```diff
@@ -70,6 +73,13 @@ class UIStore {
                 });
             }
         });
+
+        TinyDb.user.getValue(SIDEBAR_STATE_KEY).then(isOpen => {
```
Comment (anri-asaturov) [inline src/stores/ui-store.js]: Add chatSideBarIsOpen key to UIStore#prefs please, it's gonna take care of save/load and you can access it with `UIStore#prefs.chatSideBarIsOpen`
https://github.com/PeerioTechnologies/peerio-desktop/blob/fd355f6ee695856e585a0b68d03f2a6ada69916f/src/stores/ui-store.js#L21

File: src/ui/chat/ChatView.jsx [hunk 2]
```diff
@@ -197,6 +200,7 @@ class ChatView extends React.Component {
                     }
                     {this.sidebar}
                 </div>
+                {chat.leaving ? <FullCoverLoader show /> : null}
```
Comment (anri-asaturov) [inline src/ui/chat/ChatView.jsx]: 👍 😄

File: src/ui/chat/components/NoChatSelected.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+    render() {
+        if (chatStore.loading) return null;
+        return (
+            <div className="zero-message new-dm create-new-chat">
+                <div style={{
+                    marginTop: '168px',
+                    maxWidth: '600px'
+                }} >
+                    <h2 className="display-2">No chats? <strong>No problem!</strong></h2>
```
Comment (anri-asaturov) [inline src/ui/chat/components/NoChatSelected.jsx]: localization

Comment (seavan) [reply src/ui/chat/components/NoChatSelected.jsx]: this is n mvp thing, it'll get its own proper design at some point.

are you sure you want to add copy which is probably wrong?

Comment (anri-asaturov) [reply src/ui/chat/components/NoChatSelected.jsx]: ok

**Pull request merged**